File: Reactionic/src/pages/product/Product.ts
export default interface Product {
    id?: string;
    name?: string;
    detail?: string;
    unitPrice?: number;
    bundleSizes?: string; // Comma-separated string, e.g., "6,12,24"
    bundlePrice?: number;
    img?: string;
    stock?: number;
}

File: Metsys-main/src/main/java/com/example/demo/entities/Product.java
package com.example.demo.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Entity
@Table(name = "products")
@Getter @Setter
@ToString
@EqualsAndHashCode
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String detail;
    private Double unitPrice;
    private String bundleSizes; // e.g., "6,12,24"
    private Double bundlePrice;
    private String img;
    private Long stock;
}

